### PR TITLE
Follow Up : "A cool-looking box" page to be Semantically accurate

### LINKED ISSUE #15566: "A cool-looking box" exercise gives wrong impression about UI design
### MDN URL

https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/A_cool_looking_box

### What specific section or headline is this issue about?

Excercise as a whole

### What information was incorrect, unhelpful, or incomplete?

The exercise is about styling a `<p>`. The end result looks like a button and we even call it a button in the list of subtasks ("A reasonable width/height for a large button"). This gives the wrong impression that it's fine to use a styled `<p>` as a button.

### What did you expect to see?

I see two possibilities to prevent possible misunderstandings:
A) Change the styling of the box so it less looks like a button and change the mention of "button" in the subtasks.
B) Change the exercise to be about "A cool-looking button" and use `<button>` in the starting code.

Personally, I like the gradient, the shadows and the end result. I would tend to version B with no strong opinion either way.

### Do you have any supporting links, references, or citations?

_No response_

### Do you have anything more you want to share?

_No response_

### MDN metadata

<details>
<summary>Page report details</summary>

* Folder: `en-us/learn/css/building_blocks/a_cool_looking_box`
* MDN URL: https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/A_cool_looking_box
* GitHub URL: https://github.com/mdn/content/blob/main/files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md
* Last commit: https://github.com/mdn/content/commit/2777cdc8c8b54268a24beeeb7a98520e3331d3be
* Document last modified: 2022-04-26T11:32:33.000Z

</details>

### PR DESCRIPTION
**Pull request by vagadeeshwar**
#### Summary

Changed the fancy-box.png to fancy-box2.png to look more like a box instead of a button.
Tweaked instructions for accuracy.
#### Motivation

For Semantic Accuracy

#### Related issues

Fixes #15566 
Follows #15823 

#### Metadata

- [x] Adds a new document
- [x] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

File: files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md [hunk 1]
```diff
@@ -60,10 +60,10 @@ We'd like you to style the provided {{htmlelement("div")}}, giving it the follow
 
 - A reasonable width for a large box, say around 200 pixels.
 - A reasonable height for a large box, centering the text vertically in the process.
-- Centered text.
+- Horizontally Centered box.
 - A slight increase in font size, to around 17-18 pixel computed style. Use rems. Write a comment about how you worked out the value.
 - A base color for the design. Give the box this color as its background color.
-- A Contrasting color for the text; make it readable using a black text shadow.
+- A Contrasting color for the text; a black text shadow. Make it readable.
```
Comment (wbamberg) [inline files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md:66]:
```suggestion
- A contrasting color for the text and a black text shadow.
```

File: files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md [hunk 2]
```diff
@@ -60,10 +60,10 @@ We'd like you to style the provided {{htmlelement("div")}}, giving it the follow
 
 - A reasonable width for a large box, say around 200 pixels.
 - A reasonable height for a large box, centering the text vertically in the process.
-- Centered text.
+- Horizontally Centered box.
```
Comment (wbamberg) [inline files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md:63]: I'm not sure what this means. Do we mean to say: "horizontally and vertically centred text?

**Review by wbamberg — COMMENTED** [review]
This adds a new png for the box but keeps the old one, is there a reason for this?

I think it would be better, rather than a screenshot, to use a live sample for the example result, but hide all the code.

That would look something like this:

````
## Example

The finished design could look something like:

```html hidden
...the HTML does here
```

```css hidden
...the CSS goes here
```

{{EmbedLiveSample("Example")}}

````

I think we need to talk about the markup a bit too. Currently the only thing this says about the markup is:

> We'd like you to style the provided [`<div>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/div), giving it the following:

...which is wrong because the example is a `<p>`: https://github.com/mdn/learning-area/blob/main/css/styling-boxes/cool-information-box-start/index.html.

I assume the markup should be something like:

```html
<div>
  <p>This is a cool box</p>
</div>
```

...then we should be clear in the instructions about which bits of the markup we are targeting with which bits of the CSS.

For centering the text I would do something like https://codepen.io/willbamberg/pen/dydOMEd. But I'm not sure we have even taught this at this point in the course.

File: files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md (hunk 2 above)
Comment (vagadeeshwar) [reply files/en-us/learn/css/building_blocks/a_cool_looking_box/index.md:63]: Text centering has been mentioned in the previous point itself, hence for the next point i added to center the <div> using margin:0 auto

**Comment by vagadeeshwar** [comment]
I concur with providing a live sample.
About the learning-area files, i have a PR there also https://github.com/mdn/learning-area/pull/497

**Review by estelle — APPROVED** [review]

**Pull request merged**